livestream-compose-sample/src/main/java/io/getstream/livestream/compose/ui/LiveStreamCustomChannelScreen.kt

```diff
@@ -11,15 +11,15 @@ import androidx.compose.ui.unit.dp
 import io.getstream.chat.android.compose.ui.theme.ChatTheme
 
 /**
- * View component to add support for Scaffold with actionable CTA
+ * View component to add support for Scaffold with actionable CTA and main content.
  *
  * @param title - Title for the header
  * @param modifier - Modifier for styling Scaffold root component.
  * @param actions - Composable function to render actions on top bar
  * @param content - Composable function to render content below the Scaffold
  */
 @Composable
-fun CustomHeader(
+fun LiveStreamCustomChannelScreen(
```

Comment:
We specifically talked about not using a custom component for this but relying on the `Scaffold` because it already solves the problem + we don't have to expose this component and document it.

Why are we using this instead of a `Scaffold`?

Reply:
Updated

Reply:
Please advice how to shrink `setContent` here given now we have fully bloated `Scaffold` used instead of this `LiveStreamCustomChannelScreen` ( which is removed now)